.github/workflows/check-code.yml

```
@@ -46,20 +46,20 @@ jobs:
   pre-commit:
     runs-on: ubuntu-20.04
     steps:
-      - name: Checkout
+    - name: Checkout 🛎️
```

Is the indent width of this line consistent with other steps?
```suggestion
      - name: Checkout 🛎️
```